fractions/microprofile/microprofile-metrics/src/main/java/org/wildfly/swarm/microprofile/metrics/MicroprofileMetricsFraction.java

```diff
@@ -17,13 +17,13 @@
 package org.wildfly.swarm.microprofile.metrics;
 
 import org.wildfly.swarm.spi.api.Fraction;
-import org.wildfly.swarm.spi.api.annotations.Configurable;
 import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
+import org.wildfly.swarm.spi.api.annotations.WildFlyExtension;
 
 /**
  * @author Heiko W. Rupp
  */
 @DeploymentModule(name = "org.wildfly.swarm.microprofile.metrics", slot = "deployment", export = true, metaInf = DeploymentModule.MetaInfDisposition.IMPORT)
-@Configurable("swarm.microprofile.metrics")
+@WildFlyExtension(module = "org.wildfly.extension.microprofile.config")
```

Comment:
Why does the MP Metrics Fraction need to refer to MP Config WF Extension?

The presence of MP Config should be what's required to trigger the WF Extension to be loaded.

Reply:
I have no idea - I think when starting this I needed it to pull in config. May have changed since. Or I just misunderstood things. 
I guess at the time there was no overall-MP-fraction to pull in, but Metrics as standalone needed to tell that it needs MP-config.